src/HPWHHeatSources.cc

```diff
@@ -896,9 +751,15 @@ void HPWH::HeatSource::regressedMethodMP(double& ynew,
 
 void HPWH::HeatSource::btwxtInterp(double& input_BTUperHr, double& cop, std::vector<double>& target)
 {
-
-    std::vector<double> result = perfRGI->get_values_at_target(target);
-
+    std::vector<double> result;
+    try
+    {
+        result = perfRGI->get_values_at_target(target);
+    }
+    catch (std::string message)
+    {
+        hpwh->send_error(message);
+    }
```

Comment:
I believe right now it is not possible for Btwxt to throw here because of the way it has been set up. Probably don't need the try/catch.